packages/storybook/config/main.js

```diff
@@ -21,4 +21,20 @@ module.exports = {
     '@etchteam/storybook-addon-status/register',
     '@whitespace/storybook-addon-html',
   ],
+  refs: (config, { configType }) => {
+    if (configType === 'DEVELOPMENT') {
+      return {
+        vue: {
+          title: 'Component Library Vue Storybook running in development mode',
```

Comment:
Maybe it is handy to give the title the name from the sidebar, e.g. `"Vue.js Component"`, so if you wonder where the contents come from you can find it by searching for that part of the sidebar.